returnn/compile.py

```diff
@@ -24,6 +29,18 @@ def __init__(
         returnn_python_exe=None,
         returnn_root=None,
     ):
+        """
+
+        :param Path returnn_config: Path to a RETURNN config file
```

Comment:
maybe rather Path|str ?

Reply:
It is wrong in general, it should be ReturnnConfig|Path|str, the job accepts also a config object, not only a file.